api/ipfsproxy/config.go

```diff
@@ -29,32 +29,38 @@ type Config struct {
 	config.Saver
 
 	// Listen parameters for the IPFS Proxy.
-	ProxyAddr ma.Multiaddr
+	ListenAddr ma.Multiaddr
 
 	// Host/Port for the IPFS daemon.
 	NodeAddr ma.Multiaddr
 
 	// Maximum duration before timing out reading a full request
-	ProxyReadTimeout time.Duration
+	ReadTimeout time.Duration
 
 	// Maximum duration before timing out reading the headers of a request
-	ProxyReadHeaderTimeout time.Duration
+	ReadHeaderTimeout time.Duration
 
 	// Maximum duration before timing out write of the response
-	ProxyWriteTimeout time.Duration
+	WriteTimeout time.Duration
 
 	// Server-side amount of time a Keep-Alive connection will be
 	// kept idle before being reused
-	ProxyIdleTimeout time.Duration
+	IdleTimeout time.Duration
 }
 
 type jsonConfig struct {
-	ProxyListenMultiaddress string `json:"proxy_listen_multiaddress"`
+	ProxyListenMultiaddress string `json:"proxy_listen_multiaddress,omitempty"`
 	NodeMultiaddress        string `json:"node_multiaddress"`
```

Comment:
Move `NodeMultiaddress` to the new group of config fields below the line break. Also, add a comment stating that the `Proxy*` fields are there for backwards compat only.